mineral-ui/mineral-ui
# [Site] move /public back into site packge

somehow I moved `/packages/site/public` into the root of the project unknowingly. This PR puts files relevant to the website back into the site package.

### Motivation and context
It doesn't make any sense for the source of static website files to live outside the site package. It required me to write extra webpack code and sows confusion. This PR should fix this problem.

### How to test

1. `_redirects` lives in `packages/site/public`
2. `npm run build` should copy those files into `/packages/site/dist`

### Types of changes
- refactor

### Checklist




### How does this PR make you feel?
![what why](https://media.giphy.com/media/26gYERtrrzIEg6oKI/giphy.gif)
